### DISCUSSION TITLE
el-tag标签展示样式，和官方的有区别，请帮忙看看是哪里不对

### DISCUSSION THREAD

**Discussion by songday** [element-plus/element-plus#13397] [Q&A] [answered]
我参考官方文档：[Tag 标签 ](http://element-plus.org/zh-CN/component/tag.html)，写了自己的页面

我直接复制、粘贴的官方示例代码：
```
<template>
  <el-tag
    v-for="tag in dynamicTags"
    :key="tag"
    class="mx-1"
    closable
    :disable-transitions="false"
    @close="handleClose(tag)"
  >
    {{ tag }}
  </el-tag>
  <el-input
    v-if="inputVisible"
    ref="InputRef"
    v-model="inputValue"
    class="ml-1 w-20"
    size="small"
    @keyup.enter="handleInputConfirm"
    @blur="handleInputConfirm"
  />
  <el-button v-else class="button-new-tag ml-1" size="small" @click="showInput">
    + New Tag
  </el-button>
</template>
```
但是效果和官方的有些差别

官方的效果：
![1](https://github.com/element-plus/element-plus/assets/209001/099dcf3f-5395-454c-98c3-9c4736acbca8)

我的效果：
![2](https://github.com/element-plus/element-plus/assets/209001/8d7b220f-2be5-4924-b22c-62bd566882d1)

我的效果，每个tag之间，没有间距  
输入框也是100%长的，我看官方代码，没有限定输入框的长度呢？

**Comment by ryuhangyeong** [comment, marked as answer]
Hello!

[Element Plus Playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6Ijx0ZW1wbGF0ZT5cbiAgPGVsLXRhZ1xuICAgIHYtZm9yPVwidGFnIGluIGR5bmFtaWNUYWdzXCJcbiAgICA6a2V5PVwidGFnXCJcbiAgICBjbGFzcz1cIm14LTFcIlxuICAgIGNsb3NhYmxlXG4gICAgOmRpc2FibGUtdHJhbnNpdGlvbnM9XCJmYWxzZVwiXG4gICAgQGNsb3NlPVwiaGFuZGxlQ2xvc2UodGFnKVwiXG4gID5cbiAgICB7eyB0YWcgfX1cbiAgPC9lbC10YWc+XG4gIDxlbC1pbnB1dFxuICAgIHYtaWY9XCJpbnB1dFZpc2libGVcIlxuICAgIHJlZj1cIklucHV0UmVmXCJcbiAgICB2LW1vZGVsPVwiaW5wdXRWYWx1ZVwiXG4gICAgY2xhc3M9XCJtbC0xIHctMjBcIlxuICAgIHNpemU9XCJzbWFsbFwiXG4gICAgQGtleXVwLmVudGVyPVwiaGFuZGxlSW5wdXRDb25maXJtXCJcbiAgICBAYmx1cj1cImhhbmRsZUlucHV0Q29uZmlybVwiXG4gIC8+XG4gIDxlbC1idXR0b24gdi1lbHNlIGNsYXNzPVwiYnV0dG9uLW5ldy10YWcgbWwtMVwiIHNpemU9XCJzbWFsbFwiIEBjbGljaz1cInNob3dJbnB1dFwiPlxuICAgICsgTmV3IFRhZ1xuICA8L2VsLWJ1dHRvbj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyBuZXh0VGljaywgcmVmIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgRWxJbnB1dCB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxuY29uc3QgaW5wdXRWYWx1ZSA9IHJlZignJylcbmNvbnN0IGR5bmFtaWNUYWdzID0gcmVmKFsnVGFnIDEnLCAnVGFnIDInLCAnVGFnIDMnXSlcbmNvbnN0IGlucHV0VmlzaWJsZSA9IHJlZihmYWxzZSlcbmNvbnN0IElucHV0UmVmID0gcmVmPEluc3RhbmNlVHlwZTx0eXBlb2YgRWxJbnB1dD4+KClcblxuY29uc3QgaGFuZGxlQ2xvc2UgPSAodGFnOiBzdHJpbmcpID0+IHtcbiAgZHluYW1pY1RhZ3MudmFsdWUuc3BsaWNlKGR5bmFtaWNUYWdzLnZhbHVlLmluZGV4T2YodGFnKSwgMSlcbn1cblxuY29uc3Qgc2hvd0lucHV0ID0gKCkgPT4ge1xuICBpbnB1dFZpc2libGUudmFsdWUgPSB0cnVlXG4gIG5leHRUaWNrKCgpID0+IHtcbiAgICBJbnB1dFJlZi52YWx1ZSEuaW5wdXQhLmZvY3VzKClcbiAgfSlcbn1cblxuY29uc3QgaGFuZGxlSW5wdXRDb25maXJtID0gKCkgPT4ge1xuICBpZiAoaW5wdXRWYWx1ZS52YWx1ZSkge1xuICAgIGR5bmFtaWNUYWdzLnZhbHVlLnB1c2goaW5wdXRWYWx1ZS52YWx1ZSlcbiAgfVxuICBpbnB1dFZpc2libGUudmFsdWUgPSBmYWxzZVxuICBpbnB1dFZhbHVlLnZhbHVlID0gJydcbn1cbjwvc2NyaXB0PlxuXG48c3R5bGUgc2NvcGVkPlxuLm14LTEge1xuICBtYXJnaW4tbGVmdDogMC4yNXJlbTtcbiAgbWFyZ2luLXJpZ2h0OiAwLjI1cmVtO1xufVxuXG4ubWwtMSB7XG4gIG1hcmdpbi1sZWZ0OiAwLjI1cmVtO1xufVxuXG4udy0yMCB7XG4gIHdpZHRoOiA1cmVtO1xufVxuPC9zdHlsZT4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZSxcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJfbyI6e319)

The documentation of element plus is influenced by unocss. Either set the unocss of your project or specify the class yourself.

thank you

- https://unocss.dev/

**Comment by sleepyShen1989** [comment]
你可以参考下各种导入方式的[文档](https://element-plus.org/zh-CN/guide/quickstart.html#%E6%89%8B%E5%8A%A8%E5%AF%BC%E5%85%A5)，在某些情况下需要手动导入样式
